cloud

```diff
@@ -1342,6 +1344,7 @@ function cloud_functions_env() {
         [KCIDB_CLEAN_TEST_DATABASES]="$clean_test_databases"
         [KCIDB_EMPTY_TEST_DATABASES]="$empty_test_databases"
         [KCIDB_UPDATED_QUEUE_TOPIC]="$updated_topic"
+        [KCIDB_CACHE_URLS_TOPIC]="$updated_urls_topic"
```

Comment:
The environment variable name needs to follow consistent naming. It should be `KCIDB_UPDATED_URLS_TOPIC`.

Reply:
okay